### DISCUSSION TITLE
取消Mosaic数据处理

### DISCUSSION THREAD

**Discussion by 347134110** [open-mmlab/mmyolo#771] [Q&A]
进行小目标检测任务，想要取消Mosaic数据处理，该如何在train_pipeline进行修改呢？

**Comment by josh3255** [comment]
remove the dict part where mosaic is used in the config file.

**Comment by 347134110** [comment]
您好，您的邮件我已经收到了，我会尽快查看，谢谢

**Reply by josh3255** [reply]
I hope the problem will be solved :)
